### PR TITLE
Feature/cax 1 idses@idsessent 19 env for gec

### PR DESCRIPTION
**Pull request by Muravlev**
## What's new

- Dev005 landscape is created.
- Connectors deployed in dev005 landscape

## Fix
- Fixed connector helm-chart ingress
- `azurerm_resource_group.shared_services_rg` in Terraform section is a data source now instead of resource. Thus, the import of this is no longer needed. @raepple could you please review the change?

### CODE REVIEW

**Review by raepple — APPROVED** [review]

File: connector/helm-chart/templates/ingress.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ metadata:
   labels:
     {{- include "dataspace-connector.labels" . | nindent 4 }}
   annotations:
+    kubernetes.io/ingress.class: "nginx"
```
Comment (drcgjung) [inline connector/helm-chart/templates/ingress.yaml:19]: The non-beta version of ingresses (and the kubernetes version that is run in catenax) uses spec: ingressClassName: "xxx".
Furthermore we have yet setup two ingress classes "portal" and "service" in the terraform script. 
I guess that this ingress will not be correctly exposed/bound?
See https://github.com/catenax/tractusx/blob/feature/CAX-1-semantics%40CXMVP-50-semantic-adapter/infrastructure/manifests/semantics.yaml as an example.

**Review by drcgjung — CHANGES_REQUESTED** [review]
- The kubernetes version we use treats ingresses a bit different.
- The dev005 has been setup using an "old" terraform.yml which duplicated the environment names. Should be synced with main.
-

File: connector/helm-chart/templates/ingress.yaml (hunk 1 above)
Comment (Muravlev) [reply connector/helm-chart/templates/ingress.yaml:19]: I will update ingress in chart to work with relevant API version.
@raepple, could you explain what is the purpose of two ingress controllers? We have only one IP and thus only one domain name for whole landscape. Which ingress class should we use for connectors?
About ingresses - we definitely should came up with some convention (we still have only one domain name), but as far as I can see - there are no collisions. I haven't used `semantics` as the path prefix 😎
We can totally avoid collisions with another path prefix (`connectors` for example) for all connector instances, but then URI will be LARGE, e.g. https://catenaxintakssrv.germanywestcentral.cloudapp.azure.com/connectors/kaputt/consumer/

Comment (Muravlev) [reply connector/helm-chart/templates/ingress.yaml]: @drcgjung  Ingress is updated.  Please check changes

**Review by drcgjung — APPROVED** [review]

**Pull request merged**